var/spack/repos/builtin/packages/py-sphinxcontrib-bibtex/package.py

```diff
@@ -24,8 +27,12 @@ class PySphinxcontribBibtex(PythonPackage):
     depends_on('py-pybtex@0.20:', type=('build', 'run'), when='@2.0.0:')
     depends_on('py-pybtex-docutils@0.2.0:', type=('build', 'run'), when='@:1.9.9')
     depends_on('py-pybtex-docutils@0.2.2:', type=('build', 'run'), when='@2.0.0:')
+    depends_on('py-pybtex-docutils@1.0.0:', type=('build', 'run'), when='@2.2.0:')
     depends_on('py-six@1.4.1:', type=('build', 'run'), when='@0.3.5')
```

Comment:
I don't think latexcodec or six or oset are required in the latest version. Was docutils required in older versions?

Reply:
Updated. A couple less build dependencies for the latest version as a result.